Classes/Hook/ContentPostProc.php

```diff
@@ -88,23 +88,20 @@ public function main(array $params, \TYPO3\CMS\Frontend\Controller\TypoScriptFro
 
                 // No configuration found
                 if (!isset($settings['plugin.']['tx_frontendediting.'])) {
-                    $layoutPath =  'EXT:frontend_editing/Resources/Private/Layouts/';
-                    $templatePath = 'EXT:frontend_editing/Resources/Private/Templates/Toolbars/Toolbars.html';
-                    $partialPath = 'EXT:frontend_editing/Resources/Private/Partials/';
+                    $layoutPaths = ['EXT:frontend_editing/Resources/Private/Layouts/'];
+                    $templatePaths = ['EXT:frontend_editing/Resources/Private/Templates/Toolbars/Toolbars.html'];
```

Comment:
The last folder and filename of this setting should be moved; the controller name should be assigned to the rendering context and the "action" should be passed as first argument to the `render()` method.